compiler/one-cmds/one-quantize

```diff
@@ -88,6 +88,17 @@ def _get_parser():
         type=str,
         help='record mode (supported: percentile/moving_average, default=percentile)')
 
+    # arguments for force_quantparam
+    parser.add_argument(
+        '--force_quantparam',
+        action='store_true',
+        help='write quantparam to the specified tensor')
+    parser.add_argument(
+        '--tensor_name', type=str, action='append', help='tensor name (string)')
+    parser.add_argument('--scale', type=float, action='append', help='scale (float)')
+    parser.add_argument(
+        '--zero_point', type=int, action='append', help='zero point (int)')
```

Comment:
Q) if we only set one of three sub options (or two or all three), without `force_quantparam`, how does this work?
Is there any error or warning message?

From this question, I though of giving `--force_quantparam` with argument `name,scale,zero_point` in CSV format...

Which would be better?

Reply:
> if we only set one of three sub options (or two or all three), without force_quantparam, how does this work?
Is there any error or warning message?

The sub options (scale, tensor_name, zero_point) will be ignored if force_quantparam is not enabled. If the warning messages are needed, I'll add them in a separate PR (it would be better to implement in circle-quantizer).

> From this question, I though of giving --force_quantparam with argument name,scale,zero_point in CSV format...

That could be an option, but the current use case of force_quantparam is just updating the qparam of an input tensor. There is only a single (or maybe a few) target tensor at least for now, so I made the tool simply accept input tensors in command line, not a separate file.

I think we can extend force_quantparam to use a separate input file when the feature is needed (e.g., updating qparams of many tensors).

Reply:
> updating qparams of many tensors

Ah, I thought just for one tensor, `name,scale,zero_point` :)
Anyway this will require revision of whole codes in this PR... -_-;
But if you go with separate options for `name`, `zero_point` and `zero_point` the changing this to one `name,scale,zero_point` argument won't be good idea...

Reply:
Ah, I misunderstood your intention.

> Anyway this will require revision of whole codes in this PR... -_-;

Indeed.. Plus, I'm not sure `tensor_name` does not have a comma.

Reply:
> Indeed.. 

OK, let's go with current implementation..

> Plus, I'm not sure tensor_name does not have a comma.

Me too, I've seen `/`, `;` but not sure `,` -_-; ... Maybe surround whole string with `"`" :)